packages/dd-trace/test/appsec/iast/vulnerability-formatter/utils.spec.js

```diff
@@ -128,6 +128,49 @@ describe('test vulnerabiilty-formatter utils', () => {
           { start: 110, end: 112, iinfo }
         ])
       })
+
+      it('Very deep object', () => {
+        const deepObject = [...Array(100).keys()]
+          .reduce((obj) => ({ key: 'value', obj: obj || {} }), null)
+
+        const expectedObject = [...Array(50).keys()]
+          .reduce((obj) => ({ key: 'value', obj: obj || {} }), null)
+
+        const iinfo = { type: 'TEST' }
+        const deepestKey = '0'.concat('.obj'.repeat(49)).concat('.key')
+        const objRanges = {
+          [deepestKey]: [{ start: 0, end: 5, iinfo }]
+        }
+
+        const { value, ranges } = stringifyWithRanges([deepObject], objRanges)
+
+        expect(value).to.be.equal(JSON.stringify([expectedObject], null, 2))
+
+        expect(ranges).to.be.deep.equal([
+          { start: 6437, end: 6442, iinfo }
+        ])
+      })
+
+      it('Circular reference object', () => {
+        const circularObject = { key: 'value' }
+        circularObject.obj = circularObject
+
+        const expectedObject = [...Array(50).keys()]
+          .reduce((obj, i) => ({ key: 'value', obj: obj || {} }), null)
```

Comment:
Just remove the unused `i` like in "Very deep object" test.
```suggestion
          .reduce((obj) => ({ key: 'value', obj: obj || {} }), null)
```